models/step_analytics.go

```diff
@@ -7,10 +7,13 @@ import (
 
 // StepAnalytics ...
 type StepAnalytics struct {
-	StepID    string        `json:"step_id"`
-	Status    string        `json:"status"`
-	StartTime time.Time     `json:"start_time"`
-	Runtime   time.Duration `json:"run_time"`
+	StepID      string        `json:"step_id"`
+	StepTitle   *string       `json:"step_title"`
+	StepVersion string        `json:"step_verion"`
+	StepSource  *string       `json:"step_source"`
+	Status      string        `json:"status"`
+	StartTime   time.Time     `json:"start_time"`
+	Runtime     time.Duration `json:"run_time"`
```

Comment:
Please update the 
// GetTagArray ...
`func (a StepAnalytics) GetTagArray() []string` method just like you did for the `BuildAnalytics`.
https://github.com/bitrise-io/bitrise-step-analytics/blob/642b596984c77c66237052a4449ee9f09b7db539/models/step_analytics.go#L25

Reply:
Added prints.